tests/drivers/sensor/sbs_gauge/src/test_sbs_gauge.c

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <drivers/sensor.h>
+
+const struct device *get_fuel_gauge_device(void)
+{
+	const struct device *dev = DEVICE_DT_GET_ANY(sbs-gauge);
```

Comment:
```suggestion
	static const struct device *dev = DEVICE_DT_GET_ANY(sbs_gauge);
```

- `static` may not technically needed in this case, but I think it's a bit cleaner
- the change from dash to underscore is mandatory, though. How was this tested? It won't work as-is

You should be able to set up a hardware map and use `--device-testing` with twister to run these tests on the device. https://docs.zephyrproject.org/latest/guides/test/twister.html#running-tests-on-hardware